disnake/role.py

```diff
@@ -46,24 +46,37 @@ class RoleTags:
         The bot's user ID that manages this role.
     integration_id: Optional[:class:`int`]
         The integration ID that manages the role.
+
+        Roles with this ID matching the guild's ``guild_subscription`` integration
```

Comment:
thoughts on a shortcut for this? a property on the role perhaps? 

otoh we can implement that later after the api is more finalised and if there's demand

Reply:
Would be neat, but afaik there's no easy way of obtaining the `guild_subscription` integration ID without fetching integrations (which requires `manage_guild` perms).